include/ofi_mr.h

```diff
@@ -105,15 +107,21 @@ extern pthread_mutex_t mm_lock;
 
 struct ofi_mr_cache;
 
+union ofi_mr_hmem_info {
+	uint64_t reserved;
+};
```

Comment:
I'm not understanding the purpose behind this change.  Do we need to store anything outside of struct ofi_mr_info?